drivers/hmis/app/graphql/types/base_input_object.rb

```diff
@@ -20,8 +20,8 @@ def self.transform_with(transformer_class)
       @transformer = transformer_class
     end
 
-    def self.date_string_argument(name, description, **kwargs)
-      argument name, String, description, validates: { format: { with: /\d{4}-\d{2}-\d{2}/ } }, **kwargs
+    def self.date_string_argument(name, description, required:, **kwargs)
```

Comment:
Do we want required to default to false?

Reply:
yeah that makes sense. I'll update